addons/mail/static/src/owl/components/composer_text_input/composer_text_input.js

```diff
@@ -15,36 +15,20 @@ class ComposerTextInput extends Component {
      */
     constructor(...args) {
         super(...args);
-
-        /**
-         * Max frequency for searching for mention suggestions. Useful to
-         * limit amount of RPC while user types something that starts a mention.
-         */
         this.storeDispatch = useDispatch();
         this.storeGetters = useGetters();
         this.storeProps = useStore((state, props) => {
-            const composer = state.composers[props.composerLocalId];
             return {
                 isMobile: state.isMobile,
-                composer,
+                composer: state.composers[props.composerLocalId],
             };
         });
-        /**
-         * Reference of the textarea. Only useful to compute the jQuery node
```

Comment:
Good to remove the obsolete comment, but we decided to always have a comment explaining why we need a `useRef`, so it needs to be replaced with another one (in this case: setting the textarea value dynamically, and also maybe stuff related to height).

Reply:
Done